.travis.yml

```diff
@@ -1,16 +1,25 @@
 ---
+dist: trusty
```

Comment:
Precise doesn't have `shellcheck` in it's APT repos. About a month or so ago, I ran into this issue, and found that as a beta feature in Travis, you can require a Trusty-based Docker image, but must set `sudo: false` (hence, the next line).

...that said...it's still not able to find shellcheck, which I'm still looking into.